### DISCUSSION TITLE
Fedora Repository

### DISCUSSION THREAD

**Discussion by Clifra-Jones** [abraunegg/onedrive#2300] [General]
Is there any plan to get the version in the Fedora @system Repository updated to the new version?

Current Fedora 35 version is 2.4.20

**Comment by abraunegg** [comment]
@Clifra-Jones 
Unfortunately I am not responsible for the RPM packages for Fedora. You *must* raise your concern with package maintainer.

Your options to resolve:
* Email the Fedora package maintainer and get them to update the Fedora packages. Refer to:
  * https://koji.fedoraproject.org/koji/packageinfo?packageID=26044
  * https://bugzilla.redhat.com/buglist.cgi?quicksearch=onedrive
* Build the v2.4.23 RPM yourself. Refer to:
  * https://github.com/abraunegg/onedrive/blob/master/docs/build-rpm-howto.md
* Build the application from source
* Live with the obsolete messaging until the Fedora packages get updated

There is nothing I can do here to assist you.

**Comment by abraunegg** [comment]
@Clifra-Jones 
The Fedora Project has announced that Fedora 35 will reach EOL (end-of-life) on 2022-12-13. Refer to https://docs.fedoraproject.org/en-US/releases/eol/

According to Koji and Repology, v2.4.23 is now available for Fedora 36 and above:
* https://koji.fedoraproject.org/koji/packageinfo?packageID=26044
* https://repology.org/project/onedrive/versions

As you are using an EOL platform, please upgrade your version of Fedora to something that is supported.

**Comment by Clifra-Jones** [comment]
Upgraded today. Sent from my iPhoneOn Feb 17, 2023, at 3:42 PM, abraunegg ***@***.***> wrote:﻿
@Clifra-Jones
The Fedora Project has announced that Fedora 35 will reach EOL (end-of-life) on 2022-12-13. Refer to https://docs.fedoraproject.org/en-US/releases/eol/
According to Koji and Repology, v2.4.23 is now available for Fedora 36 and above:

https://koji.fedoraproject.org/koji/packageinfo?packageID=26044
https://repology.org/project/onedrive/versions

As you are using an EOL platform, please upgrade your version of Fedora to something that is supported.
